src/Illuminate/Foundation/Support/Providers/RouteServiceProvider.php

```diff
@@ -76,12 +76,19 @@ protected function loadRoutes()
 	 */
 	protected function loadRoutesFrom($path)
 	{
-		$router = $this->app['Illuminate\Routing\Router'];
```

Comment:
You have to keep this around, so that the code in `routes.php` has access to the `$router` instance (instead of having to use the facade or the helper functions).